### ISSUE TITLE
ansible docker deployment failed

### ISSUE THREAD

**Issue by caili-zhang-sas** [sassoftware/viya4-deployment#409]
I am using viya4-deployment docker container to deploy viya 4 on eks, I got the following error.
It seems that there is a `^` in password, and ansible playbook can not handle it.
### viyavars.yaml 
```
cat sasviya4aws-viyavars.yaml 
## Cluster
PROVIDER: aws
CLUSTER_NAME: jpnkcy2-eks
NAMESPACE: sasviya4aws

## MISC
DEPLOY: true # Set to false to stop at generating the manifest

#LOADBALANCER_SOURCE_RANGES: ['<cluster_nat_ip>/32']
LOADBALANCER_SOURCE_RANGES: ["xxxxxx/16", "xxxxxx/16"]
# Spexify the usual CIDR ranges assigned to SAS

## Storage - we let the tool create the SC for us
V4_CFG_MANAGE_STORAGE: true
#JUMP_SVR_RWX_FILESTORE_PATH: /volumes

## JUMP VM ACCESS TO PREPARE NFS DIRECTORIES
# JUMP_SVR_PRIVATE_KEY: '/config/jump_svr_private_key'

## SAS API Access
V4_CFG_SAS_API_KEY: 'xxxxxxxxxxxxxxxxxxxxxxxx'
V4_CFG_SAS_API_SECRET: 'xxxxxxxxxxxxxxx'
V4_CFG_ORDER_NUMBER: xxxxx

## CR Access
# V4_CFG_CR_USER: <container_registry_user>
# V4_CFG_CR_PASSWORD: <container_registry_password>

## Ingress
V4_CFG_INGRESS_TYPE: ingress
V4_CFG_INGRESS_FQDN: "jpnkcy2.gelsandbox.aws.unx.sas.com"
V4_CFG_TLS_MODE: "full-stack" # [full-stack|front-door|disabled]

## Postgres
V4_CFG_POSTGRES_TYPE: internal

## LDAP
V4_CFG_EMBEDDED_LDAP_ENABLE: true

## Consul UI
#V4_CFG_CONSUL_ENABLE_LOADBALANCER: false

## SAS/CONNECT
V4_CFG_CONNECT_ENABLE_LOADBALANCER: false

## Cadence and version
V4_CFG_CADENCE_NAME: 'stable'
V4_CFG_CADENCE_VERSION: '2023.01'

## CAS Configuration
V4_CFG_CAS_WORKER_COUNT: '3'
V4_CFG_CAS_ENABLE_BACKUP_CONTROLLER: false
V4_CFG_CAS_ENABLE_LOADBALANCER: true

# Monitoring and logging tools
V4M_BASE_DOMAIN: "jpnkcy2.gelsandbox.aws.unx.sas.com"

# allow ELASTIC SEARCH to be properly configured
V4_CFG_ELASTICSEARCH_ENABLE: true
```
### deployment command
```
$ docker container run -it --group-add root --user 1000:1000 \
   -v /home/ec2-user/project/deploy:/data \
   -v /home/ec2-user/.kube/config:/config/kubeconfig \
   -v /home/ec2-user/project/deploy/sasviya4aws/sasviya4aws-viyavars.yaml:/config/config \
   -v /home/ec2-user/viya4-iac-aws/sasviya4aws.tfstate:/config/tfstate viya4-deployment \
   --tags "baseline,viya,install"

TASK [vdm : orchestration - Download orchestration tooling image] *****************************************************************************************************
fatal: [localhost]: FAILED! => 
  msg: 'An unhandled exception occurred while templating ''xxxxxxxx^xxxxxxxxx''. Error was a <class ''ansible.errors.AnsibleError''>, original message: template error while templating string: unexpected char ''^'' at 9. String: xxxxxxx^xxxxxxxxx'
```

**Comment by jarpat** [comment]
PR is available at https://github.com/sassoftware/viya4-deployment/pull/421 & will be included in a future release.

**Comment by jarpat** [comment]
Closing, implemented in the 6.5.0 release
https://github.com/sassoftware/viya4-deployment/releases/tag/6.5.0
